Can't use the CLI on Windows

It seems, that on Windows the parent environment is not forwarded to the spawned `mvn` process, like it is on Linux.
So that I get the following error, when I try to build a simple custom war configuration:
```
Exception in thread "main" java.io.IOException: Cannot run program "mvn" (in directory "tmp\prebuild"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
	at java.lang.ProcessBuilder.start(Unknown Source)
	at io.jenkins.tools.warpackager.lib.util.SystemCommandHelper.runFor(SystemCommandHelper.java:29)
	at io.jenkins.tools.warpackager.lib.util.SystemCommandHelper.processFor(SystemCommandHelper.java:20)
	at io.jenkins.tools.warpackager.lib.util.MavenHelper.run(MavenHelper.java:60)
	at io.jenkins.tools.warpackager.lib.util.MavenHelper.run(MavenHelper.java:42)
	at io.jenkins.tools.warpackager.lib.impl.Builder.build(Builder.java:142)
	at io.jenkins.tools.warpackager.cli.Main.main(Main.java:56)
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(Unknown Source)
	at java.lang.ProcessImpl.start(Unknown Source)
	... 7 more
```
On Linux it works out of the box.

I can workaround this on windows, by calling the inner `mvn` command by myself:
```
cd tmp\prebuild
mvn package
```

--- a/custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java
+++ b/custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java
@@ -32,6 +32,7 @@ public class MavenHelper {
     private static final Logger LOGGER = Logger.getLogger(MavenHelper.class.getName());
 
     private Config cfg;
+    private static final String mvnCommand = MavenHelper.getOsSpecificMavenCommand();
 
     public MavenHelper(Config cfg) {
         this.cfg = cfg;
@@ -45,7 +46,9 @@ public class MavenHelper {
     @CheckReturnValue
     public int run(File buildDir, boolean failOnError, String ... args) throws IOException, InterruptedException {
         ArrayList<String> callArgs = new ArrayList<>();
-        callArgs.add("mvn");
+
+        callArgs.add(MavenHelper.mvnCommand);
+
         if (cfg.buildSettings != null) {
             File settingsFile = cfg.buildSettings.getMvnSettingsFile();
             if (settingsFile != null) {
@@ -61,6 +64,17 @@ public class MavenHelper {
             return 0;
         }
         return runFor(buildDir, callArgs.toArray(args));
+    }
+
+    private static String getOsSpecificMavenCommand() {
+        String mvnCmd = "mvn";
+
+        String osName = System.getProperty("os.name");
+        if(osName != null && osName.toLowerCase().contains("windows")) {
+            mvnCmd = "mvn.cmd";
+        }
+
+        return mvnCmd;
     }
 
     public boolean artifactExistsInLocalCache(DependencyInfo dep, String version, String packaging) {

